automated-comments/elixir/solution/raise_fn_clause_error.md

```diff
@@ -0,0 +1,11 @@
+<!--
+This error is raised when the submission includes a call to
+`raise FunctionClauseError` rather than having elixir fail in
+matching the call to the two_fer function for non-string types.
+-->
+
+Do not directly call `raise FunctionClauseError`.
+
+1. When elixir executes code it attempts to match [named functions](https://elixir-lang.org/getting-started/modules-and-functions.html#named-functions) call and parameters to each function from top to bottom.
+2. If you use [guards](https://hexdocs.pm/elixir/master/guards.html), it is possible to prevent functions from being invoked with arguments that do not match the functions intended use.
```

Comment:
```suggestion
2. If [guards](https://hexdocs.pm/elixir/master/guards.html) are used, it is possible to prevent functions from being invoked with arguments that do not match the functions intended use.
```

Guards may be used by code written by others or code that is "already present".

Reply:
True, I was trying to make sure the disapprove message sounded like it recommended a course of action, but I like this suggestion